index.html

```diff
@@ -181,7 +181,50 @@ <h3>Skills</h3>
                 <img id="narrow3" src="./images/light.png" alt="ico_light">
             </div>
         </div>
-    </section>   
+    </section>
+    <!--Forms section-->
+    <section>
+        <div class="ux">
+            <img id="uxlogo" src="./images/image_geometry_2.png" alt="logo">
+            <h2>Get stared</h2>
+            <p class="phrase">Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+                Voluptatum commodi aliquid quis pariatur totam explicabo
+                cupiditate, quaerat saepe dignissimos tempore nemo fugit,
+                porro illum aspernatur exercitationem rem ipsa voluptate quia.</p>
+            <form action="https://formspree.io/f/xayvddpz" method="post">
+                <ul class="data">
+                    <li class="liname">
+                        <label class="namelabel" for="name">Your name</label>
+                        <input class="name" type="text" id="name" name="user-name" placeholder="Francisco" maxlength="30" required>
+                    </li>
+                    <li class="liemail">
+                        <label class="emaillabel" for="email">Your email</label>
+                        <input class="email" type="text" id="email" name="user-email" placeholder="example@mail.com" required>
+                    </li>
+                    <li class="limsg">
+                        <textarea id="msg" name="user-message" maxlength="500" required>Enter your message here...</textarea>
+                    </li>
+                    <li>
+                        <a class ="txtbutton">Start collaboration</a>
+                    </li>
+                </ul>
+            </form>
```

Comment:
- [ ] Please add the correct `type` attribute to email input field because it's suppose to be `type="email"`.
- [ ] Please replace the `<a>` tag by a `<button>` tag with type="submit" to make the submission work for this form because now it's not working.